packages/jupyter-chat/src/widgets/chat-widget.tsx

```diff
@@ -40,7 +41,29 @@ export class ChatWidget extends ReactWidget {
 
     this._chatOptions = options;
     this.id = `jupyter-chat::widget::${options.model.name}`;
-    this.node.onclick = () => this.model.input.focus();
+    this.node.addEventListener('click', (event: MouseEvent) => {
+      const target = event.target as HTMLElement;
+      if (
+        target.closest('button, a, input, textarea, select, summary, details')
+      ) {
+        return;
+      }
```

Comment:
Thinking again about it, a more generic way could be to check if the current active element is on the chat, instead of checking if the target isl focusable element. Something like:

```suggestion
      if (this.node.contains(document.activeElement)) {
        return;
      }
```

That way we ensure that the input get focussed (and the tracker updates the current chat) only if the click occurs in a non-focusable element.

Reply:
Thanks @nakul-py for updating it quickly.